packages/lib/auth.ts

```diff
@@ -9,3 +9,19 @@ export async function verifyPassword(password: string, hashedPassword: string) {
   const isValid = await compare(password, hashedPassword);
   return isValid;
 }
+
+export function isPasswordValid(password: string) {
```

Comment:
Moved this util from `website` to `lib`, after this PR I will change `website` to use it from `lib` too.